3rdparty/assimp/assimp.cmake

```diff
@@ -16,6 +16,7 @@ ExternalProject_Add(
     CMAKE_ARGS
         -DBUILD_SHARED_LIBS=OFF
         -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
+        -DCMAKE_INSTALL_LIBDIR=${Open3D_INSTALL_LIB_DIR}
```

Comment:
Lets add this flag to:

```diff
diff --git a/3rdparty/find_dependencies.cmake b/3rdparty/find_dependencies.cmake
index dd00fd6b3..f06029d63 100644
--- a/3rdparty/find_dependencies.cmake
+++ b/3rdparty/find_dependencies.cmake
@@ -164,6 +164,7 @@ set(ExternalProject_CMAKE_ARGS
     -DCMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_CXX_COMPILER_LAUNCHER}
     -DCMAKE_CUDA_COMPILER_LAUNCHER=${CMAKE_CUDA_COMPILER_LAUNCHER}
     -DCMAKE_OSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}
+    -DCMAKE_INSTALL_LIBDIR=${Open3D_INSTALL_LIB_DIR}
     # Always build 3rd party code in Release mode. Ignored by multi-config
     # generators (XCode, MSVC). MSVC needs matching config anyway.
     -DCMAKE_BUILD_TYPE=Release
```

Since most third-party libs uses either `ExternalProject_CMAKE_ARGS` or `ExternalProject_CMAKE_ARGS_hidden`.

Reply:
I'll need to check if that works for `libjpeg-turbo`, as using it led to errors and I had to change it to `-DCMAKE_INSTALL_DEFAULT_LIBDIR=${Open3D_INSTALL_LIB_DIR}` for that one.

Reply:
Ok, I've got a new version using `ExternalProject_CMAKE_ARGS` that also works for `libjpeg-turbo`. Pushing it soon.